src/main/resources/org/sonatype/nexus/ci/iq/Messages.properties

```diff
@@ -21,6 +21,9 @@ IqPolicyEvaluation.Evaluating=Running Nexus Policy Evaluation
 IqPolicyEvaluation.AddIqServers=Add IQ Servers via:
 IqPolicyEvaluation.Stage=Stage
 IqPolicyEvaluation.Application=Application
+IqPolicyEvaluation.ManualApplication=Specify an IQ Application
+IqPolicyEvaluation.ManualApplicationVerificationFailed=Not a valid ID
```

Comment:
This is not true: IIUC an exception is thrown for not a valid ID and this is checking whether the return value is false. This is the only place a user can understand why their build/verification failed due to the reasons described: https://groups.google.com/a/sonatype.com/forum/#!topic/sonatype-insight-dev-group/Q-K7IBdVpoE.